Bottom: 10e95770a5c358fa6c2bb2f471a2fcd66cb7a2dc
Top:    e5636672e06e6efaaa908d1b2620e2918084dcd2
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 13:58:42 -0500

Refresh of ldap-add-phpldapadmin

---
diff --git a/roles/docker-openldap/defaults/main.yml b/roles/docker-openldap/defaults/main.yml
index 019799f..17b5044 100644
--- a/roles/docker-openldap/defaults/main.yml
+++ b/roles/docker-openldap/defaults/main.yml
@@ -45,7 +45,7 @@ openldap_docker_restore_dir: '{{ bacula_dest }}{{ openldap_docker_backup_dir }}'
 # Utility container - phpldapadmin
 
 # the name of the image used for the container
-phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
+phpldapadmin_image_repo: https://github.com/novatechweb/docker-phpldapadmin.git
 phpldapadmin_image_name: '{{ docker_registry_username }}/phpldapadmin'
 
 # the name of the container being started
